recipe/meta.yaml

```diff
@@ -26,9 +26,10 @@ requirements:
     - python
     - pip
     - pytest-runner
+    - setuptools_scm
 
   run:
-    - astroid >=2.5.1,<2.6
+    - astroid >=2.5.6,<2.7
     - colorama  # [win]
```

Comment:
I'd recommend dropping the win selector so this can be `noarch: python`... 
- `colorama` has no other deps
- for a development tool, it's probably not a big deal to have one or two extra packages about
- it doesn't do any freaky stuff like some of the other windows-focused packages that work "automatically"

Reply:
Switching to noarch seems worth it to me. I have a vague memory of having a problem with colorama on Linux once years ago, but I can't even remember the specifics now.